src/routes/safe/components/Transactions/TxList/TxSingularDetails.tsx

```diff
@@ -112,6 +115,16 @@ const TxSingularDetails = (): ReactElement => {
     dispatch(isTxQueued(listItemTx.txStatus) ? addQueuedTransactions(payload) : addHistoryTransactions(payload))
   }, [fetchedTx, chainId, dispatch])
 
+  if (!liveTx && error) {
+    return (
+      <FetchError
+        text="Transaction not found"
+        buttonText="Go to Queue"
+        redirectRoute={SAFE_ROUTES.TRANSACTIONS_QUEUE}
```

Comment:
Why queue? Why not to wherever `/transactions` defaults to?

Reply:
Assuming someone shared a tx with someone to sign

Reply:
That's somewhat opinionated, no? I think it's better to rely on the logic from our Router.

Reply:
Changed to TRANSACTIONS. 👍